src/ui/layout/constraints.md

```diff
@@ -1131,7 +1131,7 @@ class Example26 extends Example {
       'If all of Row\'s children are wrapped in Expanded widgets, each Expanded has a size proportional to its flex parameter, '
       'and only then each Expanded widget forces its child to have the Expanded\'s width.'
       '\n\n'
-      'In other words, Expanded ignores the preffered width of its children.';
+      'In other words, Expanded ignores the preferred width of its children.';
```

Comment:
This change needs to also be made in the matching dart code to not trigger the excerpt CI

Reply:
The matching Dart code line can be found at https://github.com/flutter/website/blob/263efceaf8a2e14c1332808983b1f70145975b3f/examples/layout/constraints/lib/main.dart#L1015

The `code-excerpt` instruction before this code block has the source filename, or you can run a global search on the adjusted to find these.

Reply:
Thanks for helping me out here!
The issue is resolved now!